engine/src/db/mod.rs

```diff
@@ -5,16 +5,17 @@ use sqlx::{migrate::MigrateDatabase, Pool, Sqlite, SqlitePool};
 pub mod handles;
 pub mod models;
 
-use crate::utils::db_path;
+use crate::utils::DB_PATH;
 
 pub async fn db_pool() -> Result<Pool<Sqlite>, sqlx::Error> {
-    let db_path = db_path().unwrap();
+    let db_path = DB_PATH.as_ref().unwrap();
```

Comment:
instead if unwrap a ? could work to.

Reply:
Because of `as_ref()` which is required to call the lazy init the type is actually `Result<_, &sqlx::Error>` and `&sqlx::Error` is not convertible into `sqlx::Error`